RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -1532,6 +1532,13 @@ void MuonIdProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   edm::ParameterSetDescription descCalo;
   descCalo.setAllowAnything();
```

Comment:
Is this line ```setAllowAnything``` still needed, given you added the relevant parameters with values below?

Reply:
I added below only the new parameters introduced in this PR, but there are many more. I do not know if it is safe to remove the `setAllowAnything`.

Reply:
Ah OK - you'd have to add all of them ...